sefaria/model/topic.py

```diff
@@ -189,6 +189,9 @@ class IntraTopicLink(abst.AbstractMongoRecord):
     optional_attrs = TopicLinkHelper.optional_attrs
     valid_links = []
 
+    def _normalize(self):
+        self["class"] = "intraTopic"
```

Comment:
I don't think you can use `[]` for `self` 
```suggestion
        setattr(self, "class", "intraTopic")
```